### PR TITLE
feat: Disabling an op replaces it with a function that throws

### PR DESCRIPTION
**Pull request by bartlomieju**
This commit changes what "disabling" an op means. Before a disabled op
was never registered. This posed some challenges, especially as we
trying to provide "virtual ops modules" (https://github.com/denoland/deno_core/issues/223),
having ops conditionally available makes it super hard to properly
snapshot and set up the runtime.

With this change, "disabling" an op makes it throw when called.

Closes https://github.com/denoland/deno_core/issues/447

### CODE REVIEW

**Comment by mmastrac** [comment]
I prefer a default of throwing rather than a no-op, as some ops might have `null` or `undefined` as a return value with special meaning and we may end up in a case where code may appear to function correctly, but was just completely non-functional.

It might even be worth replacing the op with a panicking op by default as these ops are not intended to be reachable at all.

**Comment by bartlomieju** [comment]
Discussed offline with Luca and Matt - we're gonna throw an exception from a disabled op in order not to bring the whole process down. This might change in the future.

File: core/runtime/bindings.rs [hunk 1]
```diff
@@ -493,14 +496,24 @@ fn import_meta_resolve(
...
 
+pub(crate) fn op_disabled_fn(
+  scope: &mut v8::HandleScope,
+  _args: v8::FunctionCallbackArguments,
+  _rv: v8::ReturnValue,
+) {
+  let message = v8::String::new(scope, "op is disabled").unwrap();
+  let exception = v8::Exception::error(scope, message);
+  scope.throw_exception(exception);
+}
```
Comment (mmastrac) [inline core/runtime/bindings.rs:515]: Nice!

**Review by mmastrac — APPROVED** [review]
LGTM

**Pull request merged**